packages/js/src/consts/index.ts

```diff
@@ -28,3 +28,8 @@ export enum QueryKey {
   State = 'state',
   Token = 'token',
 }
+
+export enum PromptValue {
```

Comment:
Just `Prompt`

Reply:
Aligned with the name in Kotlin SDK.
I'm good with both.

Reply:
Updated to `Prompt`